### ISSUE TITLE
Temporary solution for secure site

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/dwyl-site#413]
The site not being on HTTPS is a high priority issue, we need to fix it asap. After discussing with @iteles and @finnhodgkin we have come to the conclusion that the best thing to do right now is to use the cloudflare hack discussed in #378. According to the comments in that issue, all that needs doing is swapping the nameservers over. 

We acknowledge that this is a temporary solution, with #389 eventually being the long term solution, but due to the things discussed in #389 this will take a back seat. 

As such the immediate thing is to swap the nameservers to cloudflare as per https://github.com/dwyl/dwyl-site/issues/378#issuecomment-333887691

**Comment by markwilliamfirth** [comment]
@ZooeyMiller 

I can change the nameservers on iwantmyname, but would like to understand the reasoning behind this being high priority - as far as I am aware there is no sensitive data being submitted on the site so a secure connection is not a high priority? With regards to the google search issue - would this not be an easier temporary solution? https://support.google.com/webmasters/answer/6065812?hl=en

**Comment by ZooeyMiller** [comment]
@markwilliamfirth At the moment only emails are the only personal data being submitted, the reason for it being high priority to me, is that it demonstrates a level of care. I would not trust an agency who's own site was not even on HTTPS. HTTPS also helps with SEO (google will rank a site on HTTPS higher than one with an equivelant match otherwise which is on HTTP).

I don't know if @iteles or @finnhodgkin have more/ a different reasoning for it being high priority.

**Comment by markwilliamfirth** [comment]
@ZooeyMiller ok thanks for explaining

Nameservers have been updated

**Comment by finnhodgkin** [comment]
Just waiting on Cloudflare and then https should work:

![selection_081](https://user-images.githubusercontent.com/22300773/31181865-447727e8-a91a-11e7-9927-5c2cba875899.png)

**Comment by finnhodgkin** [comment]
The SSL is now accepted but it's not working :confused: 

Looking into this now.

**Comment by markwilliamfirth** [comment]
unrelated to above comment, but important to note:

<img width="410" alt="screen shot 2017-10-04 at 16 13 36" src="https://user-images.githubusercontent.com/5732209/31183645-148a1892-a91f-11e7-8803-e1efd69ce0c4.png">
<img width="691" alt="screen shot 2017-10-04 at 16 13 55" src="https://user-images.githubusercontent.com/5732209/31183644-147f3094-a91f-11e7-8cb7-7fa2a363e8e5.png">

**Comment by finnhodgkin** [comment]
Both those sites work for me :confused:

**Comment by markwilliamfirth** [comment]
<img width="722" alt="screen shot 2017-10-04 at 16 40 07" src="https://user-images.githubusercontent.com/5732209/31184957-b74ae57c-a922-11e7-96b4-5fc70f0dbdec.png">
<img width="717" alt="screen shot 2017-10-04 at 16 40 00" src="https://user-images.githubusercontent.com/5732209/31184958-b764945e-a922-11e7-969e-17928b8d3d05.png">
<img width="695" alt="screen shot 2017-10-04 at 16 39 52" src="https://user-images.githubusercontent.com/5732209/31184959-b76aa90c-a922-11e7-813e-365941df1797.png">

**Comment by ZooeyMiller** [comment]
They work fine for me too. Even tried on Opera (which I've only used once) and it's fine.

**Comment by finnhodgkin** [comment]
How strange :woman_shrugging:

![selection_082](https://user-images.githubusercontent.com/22300773/31185500-06491f9e-a924-11e7-92f7-b4a88f343a6a.png)

**Comment by markwilliamfirth** [comment]
How strange! Maybe just me then. Will wait and check again later

**Comment by finnhodgkin** [comment]
The https for the main site isn't working for me or @ZooeyMiller but is for you #378. Extra strange. 

Maybe the two are linked...

**Comment by finnhodgkin** [comment]
Agh yeah definitely linked. invoice.dwyl.com isn't working on my mobile but is on my desktop

**Comment by ZooeyMiller** [comment]
I',m looking into hosting with surge to see if it's any easier.

**Comment by finnhodgkin** [comment]
All DNS stuff is now handled by Cloudflare. To add subdomains just add a CNAME/A record to Cloudflare as you would with any regular nameserver. For example for invoice.dwyl.com I had to enter:

![selection_087](https://user-images.githubusercontent.com/22300773/31280419-36060bc6-aaa4-11e7-832c-ed78e9c6c15d.png)

For pages like invoice.dwyl there's one further step because the page is actually

https://docs.google.com/forms/d/e/1FAIpQLSfODeF_i5MHZPKouMHX11w7bpIbgdXmjsro7sxhDISnlk8M5w/viewform

Rather than just docs.google.com. To fix this I just added a new page rule:

![selection_088](https://user-images.githubusercontent.com/22300773/31280827-9029a828-aaa5-11e7-94e1-eb552421158d.png)

**Comment by iteles** [comment]
@finnhodgkin Can we also please document how we got over to cloudflare in the first place so that if we move back to heroku when we add #389 we can just reverse your steps?

**Comment by iteles** [comment]
Seems like the time estimate on this also needs updating.

**Comment by markwilliamfirth** [comment]
This is working now, so closing in favour of #426
